packages/docs-theme/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@blueprintjs/docs",
-  "version": "1.3.1",
+  "name": "@blueprintjs/docs-theme",
+  "version": "1.0.0",
```

Comment:
will this also go into the `latest` channel?

Reply:
yes, but I expect most sensible users to be using ranges dictated by numbers, not channel names